scripts/prepare_sector_network.py

```diff
@@ -750,6 +800,8 @@ def add_allam(n, costs):
         p_nom_extendable=True,
         # TODO: add costs to technology-data
         capital_cost=0.6 * 1.5e6 * 0.1,  # efficiency * EUR/MW * annuity
+        # TODO: add investment
+        # investment=0.6 * 1.5e6,  # efficiency * EUR/MW
```

Comment:
Correct.

Reply:
credits to CoPilot for writing that comment :D